providers/component-protocol/components/list/operation.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package list
+
+import "github.com/erda-project/erda-infra/providers/component-protocol/cptype"
+
+const (
+	// OpListPagingKey list level
+	OpListPagingKey = "changePage"
```

Comment:
changePageNo

Reply:
value is not matched with variable name.

Reply:
changePage containers changePageNo or changePageSize

Reply:
done